### ISSUE TITLE
Fixed IP or the boiler device name on your network (likely "tesy")

### ISSUE THREAD

**Issue by Pluisje69** [heggink/domoticz-tesy#1]
The line req="http://tesy/api?name=_all" is constructing a URL to send an HTTP request to the Tesy boiler. However, it's using "tesy" as the hostname instead of the IP address provided by Domoticz user interface. This might be an oversight or intentional depending on how the Tesy boiler is set up on your network. On my network it was not working and suggest to use:
req="http://"+ self.TesyIP + "/api?name=_all"

Made the following changes to get it work on my system
1.	Modified the onStart() method to use the Tesy IP provided in the plugin parameters:
req="http://" + self.TesyIP + "/api?name=_all" 
2.	Modified various req variables to use the Tesy IP provided in the plugin parameters:
req = 'http://' + self.TesyIP + '/api?name=pwr&set=' + str(nval) 
req = 'http://' + self.TesyIP + '/api?name=bst&set=' + str(nval) 
req = 'http://' + self.TesyIP + '/api?name=mode&set=' + str(nval) 
req = 'http://' + self.TesyIP + '/api?name=tmpT&set=' + str(sval) 
req = "http://" + self.TesyIP + "/api?name=_all" 
3.	Updated the onHeartbeat() method to use the Tesy IP provided in the plugin parameters:
req="http://" + self.TesyIP + "/api?name=_all"
As I am not familiar with making pyton plugins I probably overlook something.

[plugin.txt](https://github.com/heggink/domoticz-tesy/files/14623026/plugin.txt)

**Comment by heggink** [comment]
Hi Pluisje,
I think I made all teh correct changes where you suggetsed. Weirdly, I don;t use the plugin myself anymore since I was having so many mem leaks in the plugin system that I moved away from plugins as much as possible. I think most are fixed now (last 6 months, major work done by Rob (gzimocuz)). I moved to a node-red implementation that uses autodiscovery.
pls try the change to see if it works.
H
